### DISCUSSION TITLE
GitHub students benefits

### DISCUSSION THREAD

**Discussion by sonixx02** [community/community#49925] [GitHub Education] [answered]
I am 18 yrs old currently in my first year of enginnering . Currently studying at thadomal shahani College of engineering . I don't have any student I'd . But I do have all the documents to show that i do study at thadomal shahani engineering of college in information technology . But I don't know how to avail the benefits as it always show that write a student I'd but I don't have one please guide me ok how to avail these offer for students

**Comment by Anujsharma002** [comment]
use fees receipt of your college instead of id card

**Comment by sonixx02** [comment]
![Screenshot_2023-03-13-20-09-03-27](https://user-images.githubusercontent.com/112112048/224897426-43ab6d8e-b0ff-4a19-b7d4-77a152ef5443.jpg)
Please watch this there is option for college I'd only how can I upload my fee receipt

**Comment by samus-aran** [comment]
Hi @sonixx02 

Looking at your screenshot it's showing that for your selected school you need to have the academic email of `thadomal.org`, `tsec.edu`, `tsecol.onmicrosoft.com` in order to proceed.

**Reply by Anujsharma002** [reply]
i think his college is not listed or verified by github

**Comment by chmajidnaeem** [comment]
i will get by education you have a benefits for these

**Comment by anxkhn** [comment, marked as answer]
Hi @sonixx02 
Fortunately you can access Github Education via your tsecedu.org email address, you just have to replace your username with @tsecol.onmicrosoft.com and login via outlook.com for verification.

Here is a detailed article I wrote a few years ago. [This might help](https://tsecgeeks.github.io/Get-Github-Student-Pack/).
Happy to help, cheers!

**Reply by Dipeshkstha** [reply]
bro I can't switch back camera to scan my identity card. please help

**Comment by devashish2024** [comment]
@sonixx02 Seems like everything is OK, however your school is **verified** so you need to have the required email address to proceed. Consider asking your principal, she/he might had verified his/her school and that's why you need to.

**Reply by Dipeshkstha** [reply]
bro I can't switch back camera to scan my identity card. please help

**Reply by devashish2024** [reply]
Can you make new discussion and give link

**Reply by Dipeshkstha** [reply]
https://github.com/orgs/community/discussions/118532 
pls check out

**Comment by LuciferMorningstar33** [comment]
@sonixx02 You need a institutional verified email Id like [your_name@your_institution_name.edu.in]
Consider asking Your HOD or any Senior Faculty

**Comment by LuciferMorningstar33** [comment]
Mark as answer if found helpful
